project/Component/NefCompiler/Instances/NefCompilerSystem.swift

```diff
@@ -64,7 +64,6 @@ class NefCompilerSystem: CompilerSystem {
         binding(
             |<-self.buildPods(xcworkspace: xcworkspace, platform: platform, cached: cached),
             |<-self.buildCarthage(xcworkspace: xcworkspace, platform: platform, cached: cached),
-            |<-self.buildSPM(xcworkspace: xcworkspace, platform: platform, cached: cached),
```

Comment:
Why is this not build anymore?

Reply:
The reason is that Swift Package will be compiled using the swift compiler, as a project, but we do need to compile the external dependencies in a different step as we need to do in CocoaPods or Carthage.